rust/src/lib/ifaces/inter_ifaces.rs

```diff
@@ -284,16 +284,6 @@ impl Interfaces {
         ))
     }
 
-    pub(crate) fn has_sriov_enabled(&self) -> bool {
```

Comment:
Please restore this function.

Reply:
There is a warning if I added since it's no longer used at this PR, the behaviour with it is the same as with preivous nmstatectl python version, there is a timeout argument with a default value of 60s (the SRIOV timeout), so it's not needed to detect if we are at SRIOV or not.

Reply:
That function is used to determine the timeout of verification retry, not the timeout of checkpoint.

Reply:
Right, fixing.